### PR TITLE
Lens flares to WebGPU

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Review by sebavan — APPROVED** [review]

File: packages/dev/core/src/LensFlares/lensFlareSystem.ts [hunk 1]
```diff
@@ -11,19 +11,25 @@ import { Material } from "../Materials/material";
...
 import { Constants } from "../Engines/constants";
 
-import "../Shaders/lensFlare.fragment";
-import "../Shaders/lensFlare.vertex";
 import { _WarnImport } from "../Misc/devTools";
 import type { DataBuffer } from "../Buffers/dataBuffer";
 import { Color3 } from "../Maths/math.color";
 import type { Viewport } from "../Maths/math.viewport";
+import { ShaderLanguage } from "core/Materials/shaderLanguage";
+import { Observable } from "core/Misc";
```
Comment (noname0310) [inline packages/dev/core/src/LensFlares/lensFlareSystem.ts:19]:
```suggestion
import { Observable } from "core/Misc/observable";
```

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15424/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15424/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15424/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15424/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15424/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15424/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15424/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15424/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15424/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15424/merge/testResults/webgpuplaywright/index.html

**Pull request merged**